app/routes/docs.notices.unifiedschema.mdx

```diff
@@ -0,0 +1,210 @@
+              "$schema": "https://json-schema.org/draft/2020-12/schema",
+              "title": "Swift/BAT-GUANO Alert",
+              "description": "Candidate gamma-ray transient reported from the BAT-GUANO targeted search",
+              "type": "object",
+              "allOf": [
+                {
+                  "$ref": "/schema/gcn/notices/core/Alert.schema.json"
+                },
+                {
+                  "$ref": "/schema/gcn/notices/core/Event.schema.json"
+                },
+                {
+                  "$ref": "/schema/gcn/notices/core/DateTime.schema.json"
+                },
+                {
+                  "$ref": "/schema/gcn/notices/core/Localization.schema.json"
+                },
+                {
+                  "$ref": "/schema/gcn/notices/core/Statistics.schema.json"
+                },
+                {
+                  "$ref": "/schema/gcn/notices/core/Reporter.schema.json"
+                },
+                {
+                  "$ref": "/schema/gcn/notices/core/AdditionalInfo.schema.json"
+                }
+              ],
+              "properties": {
+                "$schema": true,
+                "follow_up_event": {
+                  "type": "string",
+                  "description": "Name or trigger time of the external trigger that launched the search."
+                },
+                "follow_up_type": {
+                  "type": "string",
+                  "description": "Type of external trigger that launched the search, eg GW, neutrino, etc."
+                }
+              },
+              "unevaluatedProperties": false
+            },
+            null,
+            2
+          )}
+        />
+      </Tab>
+      <Tab label="Alert.example.json">
+        <Highlight
+          language="json"
+          code={JSON.stringify(
+            {
+              "$schema": "https://gcn.nasa.gov/schema/gcn/notices/swift/bat/guano/alert.schema.json",
+              "mission": "Swift",
+              "instrument": "BAT-GUANO",
+              "messenger": "EM",
+              "record_number": 1,
+              "alert_datetime": "2023-01-01T03:24:36.0Z",
+              "alert_tense": "current",
+              "alert_type": "initial",
+              "trigger_time": "2022-12-31T21:46:05.13Z",
+              "rate_snr": 15.8,
+              "rate_duration": 0.256,
+              "rate_energy_range": [15, 350],
+              "classification": { "GRB": 1 },
+              "far": 0.00001,
+              "follow_up_event": "Fermi 694215970",
+              "follow_up_type": "GRB",
+              "data_archive_page": "https://guano.swift.psu.edu/trigger_report?id=694215995",
+              "id": ["694215995"]
+            },
+            null,
+            2
+          )}
+        />
+      </Tab>
+    </Tabs>
+    Defining structures in JSON schema can get confusing, but we are happy to help.
```

Comment:
It's good to include an offer of help here, but can we do that without saying that the process is expected to be confusing?

Reply:
I will re-word it!